app/models/folder.rb

```diff
@@ -81,13 +81,13 @@ def has_children?
   end
 
   def self.root
-    @root_folder ||= find_by_name_and_parent_id('Root folder', nil)
+    @root_folder ||= find_by_name_and_parent_id(Folder.human_attribute_name("folder.root_folder"), nil)
```

Comment:
I prefer single quotes. Could you please change that in all the `human_attribute_name` calls?